vignettes/articles/shiny.Rmd

```diff
@@ -0,0 +1,225 @@
+---
+title: "Using shiny with flexdashboard"
```

Comment:
Same with https://rmarkdown.rstudio.com/flexdashboard/shiny.html and https://github.com/rstudio/rmarkdown/blob/gh-pages/flexdashboard/shiny.Rmd ?